src/src/com/microsoft/aad/adal/AuthenticationActivity.java

```diff
@@ -282,6 +283,8 @@ public void run() {
         } else {
             Logger.v(TAG, "Reuse webview");
         }
+
+        mHelper = new StorageHelper(getApplicationContext());
```

Comment:
ok, I'll rename it.
No harm if we have initialized storage helper... It might reuse the same activity later for broker flow